fragalysis_api/xcimporter/align.py

```diff
@@ -56,23 +69,23 @@ def _get_ref(self, pdb_ref):
         Determines the best reference structure for alignments if not user provided.
         Chosen based on longest length with lowest resolution.
         :param pdb_ref: pdb to use as reference pdb for alignments
-        :return PyMol instance with reference object assigned as reference property:
+        :return: Sets the name of the __pdb_ref attribute to the align class depending on which pdb was chosen
```

Comment:
Might be also useful to return the file path of the reference as a string, in case someone wants to use this bit of code in isolation